src/Elyon.Fastly.Api.DomainServices/EmailSenderService.cs

```diff
@@ -169,6 +173,31 @@ public async Task SendInfoSessionFollowUpEmail(string receiver, string messageCo
             }).ConfigureAwait(false);
         }
 
+        public async Task SendOnboardingEmail(string receiver, IEnumerable<string> ccReceivers, int organizationTypeId, Dictionary<string, string> parameters)
+        {
+            string templateName = string.Empty;
+            List<string> attachmentsFilesHashes = new List<string>();
+            if (organizationTypeId == _companyOrganizationId)
+            {
+                templateName = "CompanyOnboarding";
+                attachmentsFilesHashes = AttachmentsByTypes.CompanyOnboarding().Select(a => a.Hash).ToList();
```

Comment:
Not a good idea to use a method that fills in content of a file into memory just to get the a small hash property out of it. You would need a second method to return you the hashes only.